src/Video/RenderProgram/StaticRenderProgram.hpp

```diff
@@ -35,6 +41,15 @@ namespace Video {
              */
             void DepthRender(Geometry::Geometry3D* geometry, const glm::mat4& viewMatrix, const glm::mat4& projectionMatrix, const glm::mat4 modelMatrix) const;
 
+            /// Render shadow pass.
+            /**
+             * @param geometry The geometry to render.
+             * @param viewMatrix The camera's view matrix.
```

Comment:
Not the camera's view matrix, the light's view matrix.

Reply:
Fixed